components/org.wso2.carbon.identity.application.authenticator.oidc/pom.xml

```diff
@@ -108,6 +108,14 @@
             <groupId>org.wso2.orbit.com.nimbusds</groupId>
             <artifactId>nimbus-jose-jwt</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.wso2.carbon.identity.inbound.auth.oauth2</groupId>
+            <artifactId>org.wso2.carbon.identity.oauth</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>com.h2database</groupId>
+            <artifactId>h2</artifactId>
```

Comment:
This is required only in test scope?

Reply:
This is used for adding the session mapping during the testing.